dockerplugin.py

```diff
@@ -198,11 +198,14 @@ def read_cpu_stats(container, dimensions, stats, t):
 
 def read_network_stats(container, dimensions, stats, t):
     """Process network utilization stats for a container."""
-    net_stats = stats['network']
+    net_stats = stats['networks']
     log.info('Reading network stats: {0}'.format(net_stats))
 
-    items = sorted(net_stats.items())
-    emit(container, dimensions, 'network.usage', [x[1] for x in items], t=t)
+    for interface, if_stats in net_stats.items():
+        items = sorted(if_stats.items())
+        interface_dims = dimensions.copy()
+        interface_dims['interface'] = interface
```

Comment:
That's a change in behavior of the metrics reporting, so we'll need a major version change for it. We'll also have to make sure that the Docker dashboards work correctly with those new timeseries.

Reply:
Dashboards looked good when I checked -- only change is the addition of an `interface` dimension, which is ignored by the built in content.  They didn't change the metric names themselves in 1.21.